kitsune/products/admin.py

```diff
@@ -16,10 +16,9 @@ def parent(obj):
         return obj.parent
     parent.short_description = 'Parent'
 
-    list_display = ('product', 'title', 'slug', parent, 'display_order',
-                    'visible')
+    list_display = ('product', 'title', 'slug', parent, 'display_order', 'visible', 'in_aaq')
     list_display_links = ('title', 'slug')
-    list_editable = ('display_order', 'visible')
+    list_editable = ('display_order', 'visible', 'in_aaq')
     list_filter = ('product', 'parent', 'slug')
```

Comment:
Since `visible` isn't one of the filter properties, I think `in_aaq` should be the same.